app/helper_function.py

```diff
@@ -0,0 +1,12 @@
+
+from flask import abort, make_response
+
+def get_valid_item_by_id(model, id):
+    try:
+        id = int(id)
+    except:
+        abort(make_response({'message': f"Invalid {model.__name__} '{id}'"}, 400))
+
+    item = model.query.get(id)
+
+    return item if item else abort(make_response({'message': f"No {model.__name__} with id {id}"}, 404))
```

Comment:
Great job creating a whole new file for to hold your helper functions! `get_valid_item_by_id` is important enough to deserve its own 💪🏻